### DISCUSSION TITLE
https://saicharankandukuri.github.io/posts/test/

### DISCUSSION THREAD

**Discussion by giscus** [SaicharanKandukuri/SaicharanKandukuri.github.io#2] [Blog]
# https://saicharankandukuri.github.io/posts/test/

Just a first post to test my blog & my plans

https://saicharankandukuri.github.io/posts/test/

**Comment by SaicharanKandukuri** [comment]
## Test

```c++
int main() {
 std::cout << "GG" << std::endl;
}
```

![GIF](https://media.giphy.com/media/26Ff21Tqjsc32wlos/giphy.gif)

**Reply by AnninoDr4** [reply]
Comments
![](https://media.giphy.com/media/k7J8aS3xpmhpK/giphy.gif)

**Reply by ayon-ssp** [reply]
![image](https://media4.giphy.com/media/3o7TKJBTMtRfSgY3yo/giphy.gif)
But I'm Enjoying
